wandi/mailer-bundle/1.0/manifest.json

```diff
@@ -0,0 +1,12 @@
+{
+    "bundles": {
+        "Wandi\\MailerBundle\\WandiMailerBundle": ["all"]
+    },
+    "copy-from-recipe": {
+        "config/": "%CONFIG_DIR%/"
+    },
+    "env": {
+        "WANDI_MAILER_SENDER_NAME": "Zinédine Zidane",
+        "WANDI_MAILER_SENDER_ADDRESS": "zinedine.zidane@france.fr"
```

Comment:
I do not think it's a good idea to set these values ​​by default. Something more generic would be better like `test@test.com`

Reply:
I agree with Max. Do a more generic example. That will make it more obvious that these values should be changes. 

After that, Im good with merging this PR.

Reply:
Ok, it's changed. Thanks.